src/DeviceInput/deviceInputSystem.ts

```diff
@@ -1,30 +1,51 @@
-import { Observable } from "../Misc/observable";
 import { Engine } from '../Engines/engine';
 import { IDisposable } from '../scene';
 import { Nullable } from '../types';
 
+/**
+ * Pointer device string
+ * @hidden
+ */
+const POINTER_DEVICE: string = "Pointer";

```

Comment:
Any reason you define these on one line, and export them on a different line? I can't see any reason not to just do:
```suggestion
export const POINTER_DEVICE: string = "Pointer";
```
And remove the export statements on lines 21-26.

Reply:
I originally did this because of a validation issue in the build process but never reverted them after just using ```@hidden```.  Changed.

Reply:
These strings have actually been removed.